hack/run-e2e.sh

```diff
@@ -24,35 +24,106 @@ source $ROOT/hack/lib.sh
 
 hack::ensure_kubectl
 hack::ensure_helm
+hack::ensure_kind
 
-E2E_IMAGE=${E2E_IMAGE:-localhost:5000/pingcap/chaos-mesh-e2e:latest}
+
+PROVIDER=${PROVIDER:-}
+CLUSTER=${CLUSTER:-}
+IMAGE_TAG=${IMAGE_TAG:-}
+E2E_IMAGE=${E2E_IMAGE:-localhost:5000/pingcap/tidb-operator-e2e:latest}
```

Comment:
Umm, do we need to change the default value?

Reply:
Did you ask why the default value of E2E_IMAGE is changed or suggest changing a new default value?

Reply:
Use a new default value for chaos mesh instead of tidb-operator? But it is not a big thing.

Reply:
This is a mistake. I didn't notice that.  updated.